### PR TITLE
Kusto Aggregate functions

### PR DESCRIPTION
**Pull request by SadiHassan**
### Changelog category (leave one):
- New Feature

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...
Below **Aggregate** functions added:
1. `make_list()`
2. `make_list_if()`
3. `make_list_with_nulls()`
4. `make_set()`
5. `make_set_if()`

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

File: src/Parsers/Kusto/KQL_ReleaseNote.md [hunk 1]
```diff
@@ -213,19 +213,19 @@ Please note that the functions listed below only take constant parameters for no
...
- - sum()
- - sumif()
- - dcount()
- - dcountif()
- - bin
\ No newline at end of file
+ - make_list()
+   `Customers | summarize t = make_list(FirstName) by FirstName`
+   `Customers | summarize t = make_list(FirstName, 10) by FirstName`
+ - make_list_if()
```
Comment (kashwy) [inline src/Parsers/Kusto/KQL_ReleaseNote.md:221]: add the link of doc for this function and all other functions m, like:
https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/makelistif-aggfunction

File: src/Parsers/Kusto/KQL_ReleaseNote.md [hunk 2]
```diff
@@ -213,19 +213,19 @@ Please note that the functions listed below only take constant parameters for no
  - tolower()  
     `Customers | project tolower(FirstName)`
 
+# July 29, 2022
```
Comment (kashwy) [inline src/Parsers/Kusto/KQL_ReleaseNote.md:216]: please sync from the Kusto-phase2 branch. (look the file inside the repo)

the Release note should put most recent on the top, not the bottom
https://github.com/ClibMouse/ClickHouse/blob/Kusto-phase2/src/Parsers/Kusto/KQL_ReleaseNote.md

there is already a pat of release date of **Augest 1, 2022**, please add to this part.

and add the link of doc for each function.

### **also do not change history**

File: src/Parsers/Kusto/KQL_ReleaseNote.md [hunk 3]
```diff
@@ -213,19 +213,19 @@ Please note that the functions listed below only take constant parameters for no
...
- - countif()
- - max()
- - maxif()
- - min()
- - minif()
- - sum()
- - sumif()
- - dcount()
- - dcountif()
- - bin
```
Comment (kashwy) [inline src/Parsers/Kusto/KQL_ReleaseNote.md:231]: do not change history

**Review by kashwy — CHANGES_REQUESTED** [review]
please update the release note

File: src/Parsers/Kusto/KQL_ReleaseNote.md (hunk 3 above)
Comment (SadiHassan) [reply src/Parsers/Kusto/KQL_ReleaseNote.md:231]: reverted back to the original

File: src/Parsers/Kusto/KQL_ReleaseNote.md (hunk 2 above)
Comment (SadiHassan) [reply src/Parsers/Kusto/KQL_ReleaseNote.md]: should be good now

File: src/Parsers/Kusto/KQL_ReleaseNote.md (hunk 1 above)
Comment (SadiHassan) [reply src/Parsers/Kusto/KQL_ReleaseNote.md]: links added

**Pull request merged**